src/rprblender/ui/object.py

```diff
@@ -48,13 +48,14 @@ def draw(self, context):
         self.layout.use_property_decorate = False
 
         rpr = context.object.rpr
+        cycles_vis = context.object.cycles_visibility
 
         flow = self.layout.grid_flow(row_major=True, even_columns=True)
-        flow.column().prop(rpr, 'visibility_in_primary_rays')
-        flow.column().prop(rpr, 'reflection_visibility')
-        flow.column().prop(rpr, 'refraction_visibility')
-        flow.column().prop(rpr, 'diffuse_visibility')
-        flow.column().prop(rpr, 'shadows')
+        flow.column().prop(cycles_vis, 'camera')
+        flow.column().prop(cycles_vis, 'reflection')
```

Comment:
Wrong Cycles property name, should be:
flow.column().prop(cycles_vis, 'glossy')